src/status_im2/common/contacts/views.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.common.contacts.views
+  (:require [status-im2.common.contacts.style :as style]
+            [i18n.i18n :as i18n]
+            [utils.debounce :as debounce]
+            [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [quo2.core :as quo2]
+            [quo2.foundations.colors :as colors]))
+
+(defn new-contact
+  []
+  (let [{:keys [public-key state error]}
+        (rf/sub [:contacts/new-identity])]
+    [rn/view
+     [rn/image {:source (js/require "../resources/images/ui/add-contact.png")}]
+     [rn/touchable-opacity
+      {:style    style/close-button
+       :on-press #(rf/dispatch [:navigate-back])}
+      [quo2/icon :i/close]]
+     [rn/view {:style style/content-outer-view}
+      [rn/view {:style style/content-inner-view}
+       [rn/view {:style style/text-combination-view}
+        [quo2/text
+         {:size   :heading-1
+          :weight :semi-bold
+          :style  {:color colors/neutral-100}}
+         (i18n/label :t/add-a-contact)]
+        [quo2/text
+         {:size   :paragraph-1
+          :weight :regular
+          :style  {:color colors/neutral-100}}
+         (i18n/label :t/find-your-friends)]]
+       [rn/view {:style style/key-view}
+        [rn/view {:style style/input-outer-view}
+         [quo2/text
+          {:size   :paragraph-2
+           :weight :medium
+           :style  {:color         colors/neutral-50
+                    :margin-bottom 8}}
+          (i18n/label :t/ens-or-chat-key)]
+         [rn/text-input
+          {:style               (style/text-input state error)
+           :placeholder         (i18n/label :t/type-some-chat-key)
+           :accessibility-label :enter-contact-code-input
+           :auto-capitalize     :none
+           :multiline           true
+           :on-change-text      #(debounce/debounce-and-dispatch
+                                  [:new-contact/set-new-identity %]
+                                  600)}]]
+        [rn/view {:style style/qr-button}
+         [rn/touchable-opacity {:on-press #()}
```

Comment:
I think we can remove the `on-press` prop if it does nothing at the moment. If functionality is added it can be added later on its corresponding PR.

Reply:
I made it a js/alert. thanks ibrahem!